src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationViewTaggerProvider.cs

```diff
@@ -92,14 +91,20 @@ protected override Task ProduceTagsAsync(TaggerContext<IClassificationTag> conte
             Debug.Assert(context.SpansToTag.IsSingle());
 
             var spanToTag = context.SpansToTag.Single();
-            var document = spanToTag.Document;
 
-            if (document == null)
+            // Attempt to get a classification service which will actually produce the results.
+            // If we can't (because we have no Document, or because the language doesn't support
+            // this service), then bail out immediately.
+            if (_classificationService == null)
```

Comment:
How would we know at the time?  The tagger is created on top of buffers/views, and may happen well before Roslyn has hooked up and we've gotten actual documents.